### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by BabyCakes13**
Restructured the probing and sensors part of the project, as follows:

top_level_architecture

### CODE REVIEW

File: homesensorhub/sensors/types/pressure.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import datetime
+
+
+class Pressure(SensorType):
+    """Class which represents a type of value collected by a sensor."""
+
+    TYPE = 'Pressure'
+
+    def get_payload(self) -> Payload:
```
Comment (alex-sever-h) [inline homesensorhub/sensors/types/pressure.py:13]: move method to SensorType parent class

File: homesensorhub/sensors/types/temperature.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import datetime
+
+
+class Temperature(SensorType):
+    """Class which represents a type of value collected by a sensor."""
+
+    TYPE = 'Temperature'
+
+    def get_payload(self) -> Payload:
```
Comment (alex-sever-h) [inline homesensorhub/sensors/types/temperature.py:13]: move method to SensorType parent class

File: homesensorhub/sensors/types/light.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    TYPE = 'Light'
+
+    def get_payload(self) -> Payload:
+        """
+        Collect payload for this sensor type.
+
+        Trigger a data collection for this type.
+        """
+        payload = Payload(self.TYPE,
```
Comment (alex-sever-h) [inline homesensorhub/sensors/types/light.py:19]: move method to SensorType parent class

File: homesensorhub/sensors/types/humidity.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+    TYPE = 'Humidity'
+
+    def get_payload(self) -> Payload:
+        """
+        Collect payload for the humidity.
+
+        Triggers a data read on the sensor for this type.
+        """
+        payload = Payload(self.TYPE,
```
Comment (alex-sever-h) [inline homesensorhub/sensors/types/humidity.py:19]: move method to SensorType parent class

File: homesensorhub/sensors/types/gas.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    TYPE = 'Gas'
+
+    def get_payload(self) -> Payload:
+        """
+        Collect payload for the gas.
+
+        Triggers a data read on the sensor for this type.
+        """
+        payload = Payload(self.TYPE,
```
Comment (alex-sever-h) [inline homesensorhub/sensors/types/gas.py:19]: move method to SensorType parent class

File: homesensorhub/sensors/types/altitude.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+    TYPE = 'Altitude'
+
+    def get_payload(self) -> Payload:
+        """
+        Collect payload for the altitude.
+
+        Triggers a data read on the sensor for this type.
+        """
+        payload = Payload(self.TYPE,
```
Comment (alex-sever-h) [inline homesensorhub/sensors/types/altitude.py:19]: move method to SensorType parent class

Comment (BabyCakes13) [reply homesensorhub/sensors/types/altitude.py:19]: *ASHAMED* :(

File: homesensorhub/sensors/types/pressure.py (hunk 1 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/types/pressure.py]: Based on this commend I ended up changing the structure again. Ty. :+1:

File: homesensorhub/sensors/types/light.py (hunk 3 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/types/light.py]: Based on this commend I ended up changing the structure again. Ty. :+1:

File: homesensorhub/sensors/types/humidity.py (hunk 4 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/types/humidity.py]: Based on this commend I ended up changing the structure again. Ty. :+1:

File: homesensorhub/sensors/types/altitude.py (hunk 6 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/types/altitude.py]: Based on this commend I ended up changing the structure again. Ty. :+1:

File: homesensorhub/sensors/types/gas.py (hunk 5 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/types/gas.py]: Based on this commend I ended up changing the structure again. Ty. :+1:

File: homesensorhub/sensors/types/temperature.py (hunk 2 above)
Comment (BabyCakes13) [reply homesensorhub/sensors/types/temperature.py]: Based on this commend I ended up changing the structure again. Ty. :+1:

**Review by alex-sever-h — APPROVED** [review]

**Pull request merged**